src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        return Colors.values()[random.nextInt(Colors.values().length)].name();
+    }
```

Comment:
It's NOT a mistake but I recommend to extract complicated expressions to separate variables to make code more easy to read
```suggestion
    public String getRandomColor() {
        int randomIndex = random.nextInt(Colors.values().length);
        return Colors.values()[randomIndex].name();
    }
```